### PR TITLE
Add trading comp page

### PR DESCRIPTION
**Pull request by TimJones97**
- Countdown timer
- List of participating users in both table and div format (desktop/mobile)
- Infinite scroll

### CODE REVIEW

**Review by dospore — COMMENTED** [review]
Am looking through it to try find root cause but as it stands it shows both the mobile nav and regular nav on desktops

File: public/scripts/theme.js [hunk 1]
```diff
@@ -20,7 +20,8 @@
     }
 })();
 
-function takePill(color) { // eslint-disable-line
+function takePill(color) {
+    // eslint-disable-line
```
Comment (dospore) [inline public/scripts/theme.js:24]: these lints may cause some errors

File: pages/_app.tsx [hunk 2]
```diff
@@ -14,7 +14,8 @@ import { useUpdateWeb3Store } from '~/hooks/useUpdateWeb3Store';
 const USERSNAP_GLOBAL_API_KEY = process.env.NEXT_PUBLIC_USERSNAP_GLOBAL_API_KEY;
 const USERSNAP_API_KEY = process.env.NEXT_PUBLIC_USERSNAP_API_KEY;
 
-const App = ({ Component, pageProps }: AppProps) => { // eslint-disable-line
+const App = ({ Component, pageProps }: AppProps) => {
+    // eslint-disable-line
```
Comment (dospore) [inline pages/_app.tsx:18]: this lint may cause errors

File: pages/_app.tsx [hunk 3]
```diff
@@ -57,6 +60,7 @@ const App = ({ Component, pageProps }: AppProps) => { // eslint-disable-line
                     src={`https://widget.usersnap.com/global/load/${USERSNAP_GLOBAL_API_KEY}?onload=onUsersnapCXLoad`}
                 />
                 <script defer data-domain="pools.tracer.finance" src="https://plausible.io/js/plausible.js" />
+                <link rel="stylesheet" href="https://use.typekit.net/klm0viv.css" />
```
Comment (dospore) [inline pages/_app.tsx:63]: is the 'aileron' font coming from issy and the designers?

File: public/scripts/theme.js (hunk 1 above)
Comment (TimJones97) [reply public/scripts/theme.js:24]: > Am looking through it to try find root cause but as it stands it shows both the mobile nav and regular nav on desktops

I made the breakpoint higher for the mobile nav to be shown to accommodate the new page link. Right now on the live Pools site, the mobile nav icon still shows on desktop (under 1280px vs under 1536px), just at a lower breakpoint:
image

File: pages/_app.tsx (hunk 2 above)
Comment (TimJones97) [reply pages/_app.tsx:18]: Will remove

File: pages/_app.tsx (hunk 3 above)
Comment (TimJones97) [reply pages/_app.tsx:63]: From Declan, yes

File: public/scripts/theme.js (hunk 1 above)
Comment (TimJones97) [reply public/scripts/theme.js]: ^ in reply to the first comment "Am looking through it to try find root cause but as it stands it shows both the mobile nav and regular nav on desktops"

Comment (dospore) [reply public/scripts/theme.js]: true, this is bad. Am pretty busy right now but should definitely be fixed up

**Review by dospore — APPROVED** [review]
lgtm

**Pull request merged**